var/spack/repos/builtin/packages/adios2/package.py

```diff
@@ -118,6 +118,10 @@ class Adios2(CMakePackage):
     # See https://github.com/ornladios/ADIOS2/pull/1899
     patch('2.5-fix-clear_cache.patch', when='@2.5.0')
 
+    def setup_build_environment(self, env):
```

Comment:
I opened an upstream issue that we can reference here:
```suggestion
    def setup_build_environment(self, env):
        # https://github.com/ornladios/ADIOS2/issues/2228
```